src/index.ts

```diff
@@ -20,8 +20,7 @@ const run = async (args: IArgs) => {
     if (botWasMention && !message.author.bot) {
       let messageContent = message.content.split(' ');
       messageContent.shift()
-      const newMessage = messageContent.join(' ');
-      const service = await Dispatcher(newMessage);
+      const service = await Dispatcher(messageContent);
```

Comment:
This does not return a service so the variable should not be called `service` It should be something like newMessage

Reply:
Cleaned up message processing logic